client/src/components/grants/Card.tsx

```diff
@@ -29,12 +30,19 @@ function Card({ projectId }: { projectId: number }) {
     }
   }, [dispatch, props.ipfsInitialized, projectId, props.currentProject]);
 
+  const checkProject = (loading: boolean, project?: Metadata) => {
```

Comment:
@schultztimothy can we maybe rename it to `getImage` or something like that since it's only about returning an image based on the loading state?